### PR TITLE
feat: login page

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
**Issue / ticket**
- #16 

**Tests covered  `__test__/pages/login`**

- [x] should render without throwing an error
- [x] should render username input
- [x] should render password input
- [x] should render the `Login` button
- [x] should accept a valid username that takes only alphanumeric characters and underscore
- [x] should not accept an invalid username that doesn't take only alphanumeric characters and underscore

Screenshot 2023-09-29 114813

https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/75125943/a83c2967-ef39-4c5b-b3c1-934b6f1859d6

### CODE REVIEW

File: __tests__/pages/login.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        usernameInput.value = 'John_doe';
+        expect(usernameInput.value).toBe('John_doe');
+    });
+    it('should not accept invalid username that doesnt take only alphanumeric characters and underscore', () => {
+        const { getByLabelText } = render(<LoginPage />);
+        const input = getByLabelText('Username');
+        fireEvent.change(input, { target: { value: 'valid_username' } });
+        expect(input).toHaveStyle('border-color: `#65a30d');
+
+        // Invalid input
```
Comment (vinit717) [inline __tests__/pages/login.js:35]: remove comment

File: src/pages/login/index.js
Comment (vinit717) [inline src/pages/login/index.js:1]: Instead of making a page, we can just modal model for login and signup

**Review by j24m — APPROVED** [review]
Looks good to merge.

**Review by iamitprakash — APPROVED** [review]

File: __tests__/pages/login.js (hunk 1 above)
Comment (gauravsinhaweb) [reply __tests__/pages/login.js]: done

File: src/pages/login/index.js
Comment (gauravsinhaweb) [reply src/pages/login/index.js:1]: discussed with @vinit717 , it is yet to be decided, future work

**Pull request merged**